base/MAPL_NUOPCWrapperMod.F90

```diff
@@ -62,6 +62,8 @@ subroutine SetServices(model, rc)
         type(ESMF_GridComp)  :: model
         integer, intent(out) :: rc
 
+        print*,"Wrapper Start SetServices"
```

Comment:
Eeep. These `print *` could be printed on every process (perhaps)? If you want to keep them, they might be better served with @tclune 's logging facility?